### DISCUSSION TITLE
Add product to the customer cart via RESP API or another approach

### DISCUSSION THREAD

**Discussion by VitaliKunts** [allegro/allegro-api#6483] [Suggestions - rejected]
Hello.

Could you please advise me how I can add many products to my shopping cart in an automated way.
Potentially it can be via REST API or any other applicable approach.

I make a lot of orders with many positions inside and doing all this stuff manually is not so convenient and time consuming.
I'll appreciate any help from your side.

Thanks in advance.

**Comment by Lukasz-Zurek** [comment]
We don't have API resources for buying purposes or from the buyer perspective right now, but thank you for your suggestion here. Thanks to this the other GitHub users can also cast their votes.

**Comment by MartaNowaczyk** [comment]
We have updated the status of the suggestion to "rejected" because we do not plan to implement the suggested changes.
